docs/en/operations/settings/settings.md

```diff
@@ -1048,6 +1048,42 @@ Result:
 └─────────────┴───────────┘
 ```
 
+## script_line_number {#script_line_number}
+
+Specifies the value for the `script_line_number` field of the [system.query_log](../system-tables/query_log.md) table.
+It improves readability of logs, and adds additional information, by indicating in which line a query is located. 
+Default value: 1.
```

Comment:
Why not 0 to indicate that it is not set?

Reply:
I will change this